src/controllers/JobRoleController.ts

```diff
@@ -3,6 +3,7 @@ import { getJobRoles } from "../services/JobRoleService";
 
 export const getAllJobRoles = async (req: express.Request, res: express.Response): Promise<void> => {
     try {
+        res.locals.loggedin = true;
```

Comment:
you can avoid having to set this new var everywhere by passing req.session.token into the view and checking if it set or not

Reply:
Added some new middleware that I think does this :) it's doing a boolean representation of req.session.token through !!req.session.token and setting this as the res.locals.loggedin value as part of every request made.